client/src/components/Events/EventCreate.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Form } from 'semantic-ui-react'
+
+class EventCreate extends React.Component {
+  state = {id: 0, title: '', description: '', startEpoch: 0, endEpoch: 0, location: '', beginner: false, tags: ''}
```

Comment:
Might be nice to extract this out of the class level as `const DEFAULT_STATE = ...` then `state = DEFAULT_STATE` inside the class level